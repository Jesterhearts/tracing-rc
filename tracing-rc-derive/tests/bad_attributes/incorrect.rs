use tracing_rc_derive::Trace;

#[derive(Trace)]
struct Fails {
    #[trace(not_an_arg)]
    item: i64,
}

fn main() {}
