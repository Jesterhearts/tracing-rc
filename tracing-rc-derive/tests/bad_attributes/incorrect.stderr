error: Only `#[trace]` and `#[trace(ignore)]` are supported on fields.
 --> tests/bad_attributes/incorrect.rs:5:5
  |
5 |     #[trace(not_an_arg)]
  |     ^
